src/lib/viewers/media/__tests__/DashControls-test.tsx

```diff
@@ -85,5 +88,23 @@ describe('DashControls', () => {
             const wrapper = getWrapper({ isPlayingHD: true });
             expect(wrapper.find(MediaSettings).prop('badge')).toEqual(<HDBadge />);
         });
+
+        describe('Subtitles', () => {
```

Comment:
We generally want to keep our test blocks in sentence structure. This would output the following, which isn't super clear:

```
DashControls render Subtitles should render SubtitlesToggle if subtitles exist
```